clients/web/src/stylesheets/widgets/accessWidget.styl

```diff
@@ -28,6 +28,21 @@
   font-weight bold
   padding 3px 10px
 
+i.g-flag-icon
+  position relative
+
+.g-flag-count-indicator
+  display inline-block
+  border-radius 3px
+  background-color rgba(200, 0, 0, 0.75)
```

Comment:
What's the rationale for transparency here? Usually I'd expect an overlay like this to be opaque.